modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Alternative.kt

```diff
@@ -46,4 +46,6 @@ interface Alternative<F> : Applicative<F>, MonoidK<F> {
    * @returns a combination of both computations.
    */
   fun <A> Kind<F, A>.orElse(b: Kind<F, A>): Kind<F, A>
+
+  override fun <A> Kind<F, A>.combineK(y: Kind<F, A>): Kind<F, A> = orElse(y)
```

Comment:
wat


that simple?

Reply:
Well yes. orElse and empty form a monoid, that is the very reason it is a monoidk in the first place. Alternative laws are usually just testing if orElse is associative and if empty is the identity. We have a lot more laws on it's interactions with applicatives, but in essence alternative is just a monoid with orElse and empty (over kinds that's why it's MonoidK).